document/core/exec/instructions.rst

```diff
@@ -2002,49 +2004,43 @@ When the end of a block is reached without a jump or trap aborting it, then the
    pair: handling; exception
 
 .. _exec-catchn:
-.. _exec-exn-handling:
 
 Exception Handling
 ~~~~~~~~~~~~~~~~~~
 
-The following auxiliary rules define the semantics of handling thrown exceptions
-inside :ref:`throw contexts <syntax-ctxt-throw>`, such as installing, searching for,
-and exiting an :ref:`exception handler <syntax-handler>` :math:`\CATCHN_n`.
-
-.. _exec-install-handler:
+The following auxiliary rules define the semantics of entering and exiting exception handlers through :ref:`try <syntax-try>` instructions and handling thrown exceptions.
 
-Installing an exception handler :math:`H` for a block :math:`\instr^\ast` with label :math:`L`
-..............................................................................................
+.. _exec-handler-enter:
 
-1. Let :math:`H` be the exception handler :math:`\CATCHN_m{\instr'^\ast}`.
+Entering an exception handler :math:`H`
+.......................................
 
-2. Push :math:`H` on the stack.
-
-3. :ref:`Enter <exec-instr-seq-enter>` the block :math:`\instr^\ast` with label :math:`L`.
+1. Push :math:`H` onto the stack.
 
 .. note::
    No formal reduction rule is needed for installing an exception handler
    because it is an :ref:`administrative instruction <syntax-instr-admin>`
-   which exception throwing instructions will search for.
+   that the :ref:`try <syntax-try>` instruction reduced to directly.
 
+.. _exec-handler-exit:
 
-Exiting an exception handler with arity :math:`m` from a block with label :math:`L`
-...................................................................................
+Exiting an exception handler
+............................
 
-When the block is exited without a throw :ref:`triggering <exec-handle-exn>`
+When the end of a :ref:`try <syntax-try>` instruction is reached without a jump, exception or trap, then the following steps are performed.
```

Comment:
I just noted that a similar language tweak is probably necessary to the intro sentences to the sections on exiting a block and returning from a function (including exceptions in the enumerations of possibilities).